tendrl/bridge_common/tests/sds_state_gluster.yaml

```diff
@@ -0,0 +1,151 @@
+---
+tendrl_schema_version: 1
+valid_objects:
+- tendrl_context
+- volume
+- brick
+- peer
+
+object_details:
+  tendrl_context:
```

Comment:
not all the fields added here. the list of fields is cluster_id, sds_name and sds_version as per latest format.